docs/minio-client-complete-guide.md

```diff
@@ -507,15 +509,20 @@ ENVIRONMENT VARIABLES:
 
 *Example: Stream MySQL database dump to Amazon S3 directly.*
 
-```sh
+```
 mysqldump -u root -p ******* accountsdb | mc pipe s3/sql-backups/backups/accountsdb-oct-9-2015.sql
 ```
+*Example: Copy an ISO image to an object on Amazon S3 cloud storage.*
```

Comment:
I am not sure when we are adding this example, we still have the one above (mysqldump)

`cat debian.iso | mc pipe alias/bucket/debian.iso` can be easily replaced by `mc cp debian.iso alias/buckt/debian.iso`, mysqldump makes more sense here (upload mysqldump output to the S3 server on the fly)

Reply:
Removed `cat debian.iso | mc pipe alias/bucket/debian.iso `  example.